### ISSUE TITLE
{{#eq}} has empty context starting with v0.9.0

### ISSUE THREAD

**Issue by warncke** [helpers/handlebars-helpers#279]
In the included template file the {{#eq}} blocks lose all context data. I tried doing a {{jsonStringify this}} before and inside the {{eq}} block and it goes from being the expected data to an empty object.

This was working in 0.8.4 but not working in 0.9.0+

```
<div class="pure-u-1 ">
    {{#if field.legend}}
        {{>form/field-legend}}
    {{else if field.nested}}
        {{#if field.array}}
            {{>form/field-nested-array}}
        {{/if}}
    {{else}}
        {{#eq field.inputType 'text'}}
            {{>form/field-text}}
        {{/eq}}
        {{#eq field.inputType 'textarea'}}
            {{>form/field-textarea}}
        {{/eq}}
        {{#eq field.inputType 'password'}}
            {{>form/field-password}}
        {{/eq}}
        {{#eq field.inputType 'checkbox'}}
            {{>form/field-checkbox}}
        {{/eq}}
        {{#eq field.inputType 'select'}}
            {{>form/field-select}}
        {{/eq}}
        {{#eq field.inputType 'hidden'}}
            {{>form/field-hidden}}
        {{/eq}}
        {{#eq field.inputType 'color'}}
            {{>form/field-color}}
        {{/eq}}
    {{/if}}
</div>

**Comment by doowb** [comment]
Thanks for the issue, I think I see what the problem is.

**Comment by doowb** [comment]
@warncke this should be fixed in `0.9.3`. Feel free to reopen if you're still running into this issue.

**Comment by warncke** [comment]
Yep, looks good. Thanks!
